libraries/forms.py

```
@@ -368,21 +368,19 @@ def core_batchable(x):
             mailing_list_contributors_task.get()
         )
-        (mailinglist_post_stats, total_mailinglist_count) = (
-            mailing_list_stats_task.get()
-        )
-        (commit_contributors_release_count, commit_contributors_new_count) = (
+        mailinglist_post_stats, total_mailinglist_count = mailing_list_stats_task.get()
+        commit_contributors_release_count, commit_contributors_new_count = (
             commit_contributors_task.get()
         )
         (
             mailinglist_words,
```

Ruff (0.15.15)

[warning] Unpacked variable `mailinglist_words` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)